overrides/android-override.json

```diff
@@ -1313,5 +1313,13 @@
                 }
             }
         ]
+    },
+    "changeOmnibarPosition": {
+        "state": "internal",
+        "features": {
+            "refactor": {
+                "state": "disabled"
+            }
+        }
```

Comment:
@0nko, where is the `changeOmnibarPosition` and `refactor` flags used in the Android code. We already have an `androidBrowserConfig` feature, maybe we just add a sub-feature in threre?

Reply:
@aitorvs It’s used [here](https://github.com/duckduckgo/Android/pull/4885/files#diff-afc0d0350ec07d2ca8049fc263053a34faf4841263abc688f8c65294ede7be68). The `refactor` feature will be used by @malmstein in a subsequent refactoring of the omnibar. I don’t have any preference here, David suggested to make these changes so I’ll let him decide what’s better.